src/main/java/com/backend/soullive_a/entity/ModelRecentWork.java

```diff
@@ -0,0 +1,28 @@
+package com.backend.soullive_a.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+
+public class ModelRecentWork {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "MODEL_RECENT_WORK_ID", nullable = false)
+    private Long id;
+
+    private Integer year;
```

Comment:
```suggestion
    private Integer year;

```
한줄 띄우는거 모든 엔티티 다 해주시고 nullable 다 달아주세요..!